### ISSUE TITLE
graphical abstract

### ISSUE THREAD

**Issue by raynamharris** [gvwilson/collaborative-lesson-development#73]
Replaces https://github.com/gvwilson/collaborative-lesson-development/issues/23.I'd appreciate your feedback and suggestions for improve. 

Two things: 1) I may have a little overboard with the color. 2) The "you have died peer review" is probably not appropriate, but I do think its funny"

**Comment by raynamharris** [comment]
![2017-05-17-rules-02](https://cloud.githubusercontent.com/assets/7544197/26732050/d741d75e-477b-11e7-8350-1aecc4f6eace.png)

**Comment by gvwilson** [comment]
:+1: except I don't get the "you have died of peer review" reference - is that a meme?

**Comment by raynamharris** [comment]
Yes, its a meme. There's a computer game from the 80s/90s with a "You have died of dysentery" meme. A year or so ago someone made this image and sold it on t-shirts for ASAPbio.

**Comment by raynamharris** [comment]
Okay, now without a meme but still lots of emoji-like characters. This one I'm pretty happy with.
![2017-05-17-rules-02](https://cloud.githubusercontent.com/assets/7544197/26737256/f55bccdc-478e-11e7-9d97-84d45ddc2339.png)
